rwengine/src/engine/GameData.cpp

```diff
@@ -341,22 +341,12 @@ void GameData::loadHandling(const std::string& path)
 	l.loadHandling(path, vehicleInfo);
 }
 
-SCMFile *GameData::loadSCM(const std::string &path)
+SCMFile *GameData::loadSCM(const std::string &name)
 {
-	std::ifstream f(datpath + "/" + path);
-
-	if(! f.is_open() ) return nullptr;
-
-	f.seekg(0, std::ios_base::end);
-	unsigned int size = f.tellg();
-	f.seekg(0);
-
-	char* buff = new char[size];
-	f.read(buff, size);
+	auto scm_h = openFile(name);
```

Comment:
nit: Dan likes to camelCase variable names.

(as this is the only thing I can nit about this is fine to me, we'll probably catch it later when loading SCM from the proper path with a better IO abstraction)